src/components/mypage/ReviewModal/WriteReviewModal.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, Story } from '@storybook/react'
+import type { WriteReviewModalProps } from './WriteReviewModal /types'
```

Comment:
```suggestion
import type { WriteReviewModalProps } from './WriteReviewModal/types'
```
WriteReviewModal 디렉토리에 띄어쓰기가 포함되어 있어서 수정해서 올려야할 것 같습니다!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/108/commits/d45593796d2ec7c49c4161da3621af7d44fb3859)